index.js

```diff
@@ -113,3 +115,35 @@ export async function extendFromSystemPath(languagesDirPath) {
     }
   }
 }
+
+export async function fetchTranslations(repos) {
+  await Promise.all(repos.map(async(repoData) => {
+    const { repo, branch = "main" } = repoData;
+
+    if (typeof repo !== "string" || repo.trim() === "") {
+      throw new Error("The repo must be a non-empty string.");
+    }
+
+    if (!await checkRepositoryExistence(repo)) {
+      throw new Error(`Repository NodeSecure/${repo} does not exist.`);
+    }
+
+    const { location } = await github.downloadAndExtract(`NodeSecure.${repo}`, { branch });
+
+    const i18nPath = path.join(location, "i18n");
+
+    fs.access(i18nPath, fs.constants.F_OK, (err) => {
+      if (err) {
+        throw new Error(`NodeSecure/${repo} does not have a /i18n folder`);
+      }
+    });
+
+    await extendFromSystemPath(i18nPath);
+  }));
+}
+
+async function checkRepositoryExistence(repoName) {
+  const response = await fetch(`https://api.github.com/repos/NodeSecure/${repoName}`);
+
+  return response.status === 200;
+}
```

Comment:
We could check `response.data.default_branch` is equal to the given branch (or `main` by default)
It would throws for `scanner` & `cli` if we don't set `branch: master`.

Reply:
Well noted, I take that into account

Reply:
(it may be useless as this is not an API but a script)